### DISCUSSION TITLE
Source for "toga-demo"

### DISCUSSION THREAD

**Discussion by White-Rabbit-Scientific** [beeware/toga#1694] [Q&A]
I was playing with "toga-demo" and noted that on Linux there is a menu leading to a Credits button:

Help ---> About Toga Demo ---> Credits

which has a cool animation (on Linux, not on Windows).

I cannot figure how this was produced (the word "Credits" does not appear in the code anywhere)? 

Is the Credits thing purely Python, or achieved from toml files and other cool trickery?

**Comment by freakboy3742** [comment]
The credits button is a feature of the "About" dialog provided by the underlying GTK toolkit. 

The content of that dialog is [populated automatically by Toga](https://github.com/beeware/toga/blob/main/gtk/src/toga_gtk/app.py#L208), reading from the app's metadata.
